f5/bigip/mixins.py

```diff
@@ -108,3 +108,52 @@ def __setattr__(self, key, value):
                     [self.__dict__.pop(n, '') for n in new_set]
         # Now set the attribute
         super(ExclusiveAttributesMixin, self).__setattr__(key, value)
+
+
+class UnnamedResourceMixin(object):
+    '''This makes a resource object work if there is no name.
+
+    These objects do not support create or delete and are often found
+    as Resources that are under an organizing collection.  For example
+    the `mgmt/tm/sys/global-settings` is one of these and has a kind of
+    `tm:sys:global-settings:global-settingsstate` and the URI does not
+    match the kind.
+    '''
```

Comment:
I just checked, we don't support `load` on anything but `Resource`.   I think this is correct, I think `load` is like `create` and `delete` in that it is only really intended to be used with server-resources that have unique elements in their URI-path.